ouroboros-consensus/test/consensus-test/Test/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -497,6 +498,7 @@ runChainSync skew securityParam (ClientUpdates clientUpdates)
                  chainSyncTracer
                  chainDbView
                  varHandles
+                 (pure Syncing)
```

Comment:
Maybe leave a comment this this only affects the LoP bucket, which is disabled here, so the choice is irrelevant?

Reply:
Done in 3890e9bf6b0df74d3a0faff17f7beb703b16a799.